src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,124 @@
+"""
+    NormInfinityBridge{T}
+
+The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\max_i |x_i|`` if and only if ``t \\ge x_i`` and ``t \\ge -x_i`` for all ``i``.
+"""
+struct NormInfinityBridge{T} <: AbstractBridge
+    nn_index::CI{<:Union{MOI.VectorAffineFunction{T}, MOI.VectorQuadraticFunction{T}}, MOI.Nonnegatives}
```

Comment:
You should infer the type of the function in `concrete_bridge_type`, see
https://github.com/JuliaOpt/MathOptInterface.jl/blob/af426f0f96142609ab18ee1ba91c6b4ca425a50c/src/Bridges/Constraint/geomean.jl#L108-L116

Reply:
OK I'll have a go at this though I'm still a little confused. 
How in the GeoMeanBridge code can we sometimes have `GeoMeanBridge{T}` and sometimes `GeoMeanBridge{T, F}` and sometimes `GeoMeanBridge{T, F, G}`?

Reply:
I mean does Julia just allow us to lop off the last type parameters?

Reply:
When you do `::GeoMeanBridge{T}` for an argument, you allow the second and third type parameters to be anything and you won't reference them in the function.
It is equivalent to doing `::GeoMeanBridge{T, F, G} ... where {F, G}` but when you don't use `F` and `G` in the function, I prefer just writing `::GeoMeanBridge{T}`.
For `::Type{<:GeoMeanBridge{T}}`, it's the same.